[biew] Unable to install

```
make: *** No rule to make target 'biewlib/sysdep/arm/aclib.c', needed by 'biewlib/sysdep/arm/aclib.o'.  Stop.
make: *** Waiting for unfinished jobs....
biewlib/tw_class.c: In function ‘twcFindClass’:
biewlib/tw_class.c:74:11: warning: assignment discards ‘const’ qualifier from pointer target type
  key.name = name;
           ^
biew failed to build: `make V=0 -j4` exited with 2
```

--- a/packages/biew.rb
+++ b/packages/biew.rb
@@ -3,8 +3,8 @@ require 'package'
 class Biew < Package
   description 'EYE (Binary EYE) is a free, portable, advanced file viewer with built-in editor for binary, hexadecimal and disassembler modes.'
   homepage 'https://sourceforge.net/projects/beye/'
-  version '6.1.0'
-  source_url 'http://downloads.sourceforge.net/project/beye/biew/6.1.0/biew-610-src.tar.bz2'
+  version '6.1.0-0'
+  source_url 'https://downloads.sourceforge.net/project/beye/biew/6.1.0/biew-610-src.tar.bz2'
   source_sha256 '2e85f03c908dd6ec832461fbfbc79169a33f4caccf48c8fe60cbd29f5fb06d17'
 
   binary_url ({
@@ -21,6 +21,7 @@ class Biew < Package
   depends_on 'slang'
 
   def self.build
+    system "cp -r ./biewlib/sysdep/generic ./biewlib/sysdep/arm" # add arm support
     system "./configure --prefix=#{CREW_PREFIX}"
     system "make"
   end

